src/count-loc.ts

```diff
@@ -0,0 +1,64 @@
+import { LocDir } from "github-linguist";
+
+import { Logger } from "./logging";
+
+const supportedLanguages = {
+  c: "cpp",
+  "c++": "cpp",
+  "c#": "csharp",
+  go: "go",
+  java: "java",
+  javascript: "javascript",
+  python: "python",
+  ruby: "ruby",
+  typescript: "jypescript",
```

Comment:
Awkwardly, although the CodeQL language identifiers are `javascript` and `python`, the conventional prefixes we appear to have adopted for query IDs targeting these languages is `js/` and `py/`... Easier to handle that here.

```suggestion
  javascript: "js",
  python: "py",
  ruby: "ruby",
  typescript: "js",
```

Reply:
and apparently `csharp` -> `cs`